CfStreamUploader/CfStreamUploader.Presentation/ViewModels/MainViewModel.cs

```diff
@@ -364,6 +369,11 @@ internal bool IsConfigSolid()
                    this.Core.ConfigManager.Config.UserSettings.CfAccount != string.Empty;
         }
 
+        private void OpenHistory()
+        {
+            this.Core.HistoryManager.CreateVideoUploadHistoryFolder();
+            this.Core.HistoryManager.OpenVideoUploadHistoryFolder();
```

Comment:
Move CreateVideoUploadHistoryFolder in OpenVideoUploadHistoryFolder --> 1 method

rename: CreateVideoUploadHistory, OpenVideoUploadHistory